app/components/base/editor/editor.component.ts

```diff
@@ -83,6 +95,10 @@ export class EditorComponent implements ControlValueAccessor, AfterViewInit, OnC
             this.blur.emit();
             this.changeDetector.markForCheck();
         });
+
+        setTimeout(() => {
```

Comment:
Shouldn't you be discarding this timeout when you navigate away from the component?